modules/core/src/lib/attribute-transition-utils.js

```diff
@@ -20,6 +20,9 @@ export function normalizeTransitionSettings(userSettings, layerSettings) {
     userSettings = {duration: userSettings};
   }
   userSettings.type = userSettings.type || 'interpolation';
+  if (userSettings.type === 'interpolation' && userSettings.duration < 1) {
+    return null; // return null because there's no transition to perform
+  }
```

Comment:
Feels a bit weird to return `null` from a `normalizeTransitionSettings()` function that has been supplied a valid transition setting, but I guess this is what we were doing before.

Reply:
I'm handling this inside `GPUInterpolationTransition` in https://github.com/uber/deck.gl/pull/3649